orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -344,6 +353,12 @@ class ORTTrainerOptions(object):
             can be specified by extending :py:class:`.LossScaler` class from scratch
         graph_transformer (dict):
             graph transformer related configurations
+        graph_transformer.attn_dropout_recompute(bool, default False)
+        graph_transformer.gelu_recompute(bool, default False)
+        graph_transformer.transformer_layer_recompute(bool, default False)
+        graph_transformer.number_recompute_layers(bool, default False)
+        graph_transformer.propagate_cast_ops_level(integet, default -1)
+        graph_transformer.propagate_cast_ops_allow(list of str, [])
```

Comment:
Although I am pointing out this on ORTTrainer side, this is valid to ORTModule too. This doesnt add any extra info what these flags are for and what they do